test/support/utils_test_base.h

```diff
@@ -406,7 +406,7 @@ void update_data(TTestDataTransfer& helper, Args&& ...args)
     static constexpr float ScaleMax = ScaleMaxNValue;                                                             \
     static constexpr float ScaleStep = ScaleStepValue;
 #else
-#define DEFINE_TEST_CONSTRUCTOR(TestClassName, ScaleMaxNValue, ScaleStepValue)
```

Comment:
These changes seem to be benign changes unrelated to the PR.  I'd happily approve them separately, but they probably don't belong here. If you want to make this PR rebased down to 2 or 3 commits, with these separated, that would be find with me.

Reply:
Reverted.